### DISCUSSION TITLE
AI對職業的影響

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2079] [General]
https://tw.news.yahoo.com/ai%E4%BE%86%E8%A5%B2-%E6%9C%AA%E4%BE%8610%E5%B9%B4-%E9%80%99%E8%81%B7%E6%A5%AD-%E6%9C%80%E6%85%98%E6%B8%9B%E5%B9%8538-%E7%BE%8E%E9%A0%90%E6%B8%AC-132013242.html
在現今科技越來越進步的同時，AI帶給我們的助益越來越多，然而其同時也帶來了一些負面影響，例如職業的減少，如文章中提及的文字處理員與打字員等，時代所趨，我認為我們應不斷精進自身能力，而非坐以待斃，等待他人的進步以改變自身條件，減少怨天尤人的時間，努力地去學習更多新知，才是當今的我們應努力做到的。

**Comment by peresonacifer** [comment]
人工智慧的發展已經是時代的趨勢，隨著時間的發展，AI的能力只會越來越強大，當然會給我們的生活帶來極大的便利，但相對的在工作上面，一定會取代很多職位，如果想不被取代，能做的就是與時俱進，並且培養更多的專長，但多虧了AI，我能學習的東西變得更多且更快，所以在努力的過程中，一定要多接觸AI。

**Comment by ds0w0** [comment]
我認為我們身在一個需要不斷學習和成長的時代，每個人都應該積極適應並把握這些變化帶來的機遇。隨著科技的快速進步像是AI的發展確實帶來了許多便利，但也同時對於純粹技術或工作機械化的職業構成威脅。時代的變遷是不可避免的，只有積極應對、學習新的技術和知識，保持擁有決策和與人交流的能力，才能在變化的環境中找到新的機會，保持競爭力。
